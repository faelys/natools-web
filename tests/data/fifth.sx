(elements
   (main-text "\
<p>Data in the fifth page.</p>\n"
      (if-parameter-is extra yes
         "<p>With an extra paragraph, as requested.</p>\n")
(render page-comments))
   (title "Fifth Page"))
(tags (5 pages/news))
(comments
   (default-text-filter html-escape)
   (flags ignore-by-default allow-date-override allow-ignore)
   (filter comment-filter)
   (backend transient-comments 5)
   (tags comments)
   (text-filters pass-through))
