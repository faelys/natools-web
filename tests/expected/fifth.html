<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
<html>
  <head>
    <title>Fifth Page</title>
  </head>
  <body>
<h1>Fifth Page</h1>
<p>Data in the fifth page.</p>
<h1>Comments</h1>
<ol>
</ol>
<h1 id="newcomment">Post a Comment</h1>
<form class="mainform" method="post" action="/fifth/comments">
  <p><label>Name or nick:
    <input name="c_name" value="" type="text">
  </label></p>
  <p><label>E-mail address:
    <input name="c_mail" value="" type="text">
  </label></p>
  <p><label>Web site:
    <input name="c_link" value="" type="text">
  </label></p><!--
  <p><label>Very special field:
    <input name="address" value="http://" type="text">
  </label></p>-->
  <p><label>Comment text:
    <textarea name="c_text" cols="80" rows="10"></textarea>
  </label></p>
  <p class="buttons">
    <input value="Preview" name="preview" type="submit">
    <input value="Submit" name="submit" type="submit">
  </p>
</form>
    <div class="sidebar">
<h1>Side bar</h1>
<p>Tags:</p>
<p>Previous: <a href="/fourth">Fourth Page</a></p>
    </div>
    <div class="footer">
<p>Default footer: <a href="/index">return to index</a></p>
    </div>
  </body>
</html>
